### DISCUSSION TITLE
Looking for "Starstruck"? 🌟

### DISCUSSION THREAD

**Discussion by hitblast** [Schweinepriester/github-profile-achievements#353] [General]
I'm currently developing a Discord bot with the mission of providing its end-users with a neat solution of unified commands. If you like this idea, consider **dropping a star** below -

https://github.com/IgKniteDev/IgKnite

and I'll star you back as well. You might also wanna consider dropping the link to your GitHub Repository down in the replies section below.

**Comment by Ashu-Barnwal** [comment]
DONE
https://github.com/Ashu-Barnwal/Color-Plate
Thank you!!

**Reply by hitblast** [reply]
Done as well!

**Reply by aarongeo1** [reply]
@Ashu-Barnwal done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@Ashu-Barnwal I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Raz-js** [comment]
Done

**Reply by hitblast** [reply]
Thanks! If you want then you can mention one of your important repositories here :P

**Comment by anarchy1337x** [comment]
Done.
https://github.com/anarchy1337x/socks5-proxy-checker

**Reply by hitblast** [reply]
Thanks! Done as well.

**Reply by PatrickDSiqueira** [reply]
@anarchy1337x  I stared you repository too

my repository is 
https://github.com/PatrickDSiqueira/BuscadorCEP

**Reply by WHAHA-HA** [reply]
@anarchy1337x, done
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by aarongeo1** [reply]
@anarchy1337x done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@anarchy1337x I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@PatrickDSiqueira I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by polekstulod** [comment]
Done. help me too 🥺 https://github.com/polekstulod/foodtrip-api

**Reply by hitblast** [reply]
Done :D

**Reply by aarongeo1** [reply]
@polekstulod done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@polekstulod I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by ghostinthematrix0** [comment]
https://github.com/mendeeislife/mendeeislife

**Reply by hitblast** [reply]
Starred. Hope you'll help out as well :D

**Reply by WHAHA-HA** [reply]
@mendeeislife, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by aarongeo1** [reply]
@mendeeislife done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@mendeeislife I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by deadnews** [comment]
Done.
https://github.com/DeadNews/mc-onedark

**Reply by hitblast** [reply]
@DeadNews Thanks! Done as well.

**Reply by miladkhalighi** [reply]
Done 
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by WHAHA-HA** [reply]
@DeadNews, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@DeadNews I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by PatrickDSiqueira** [comment]
done 
https://github.com/PatrickDSiqueira/BuscadorCEP

**Reply by hitblast** [reply]
Done as well!

**Reply by PatrickDSiqueira** [reply]
Thank you

**Reply by miladkhalighi** [reply]
Done
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by kshitij-halankar** [reply]
@PatrickDSiqueira Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by aarongeo1** [reply]
@PatrickDSiqueira done https://github.com/aarongeo1/NUM-CLASSIFY

**Comment by miladkhalighi** [comment]
Done
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by hitblast** [reply]
Done as well :D

**Reply by WHAHA-HA** [reply]
@miladkhalighi, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by miladkhalighi** [reply]
@WHAHA-HA done

**Reply by aarongeo1** [reply]
@WHAHA-HA done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@miladkhalighi I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Reply by hitblast** [reply]
Thanks! Giving a star as well.

**Reply by aarongeo1** [reply]
@danielr1996 done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@danielr1996 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by kshitij-halankar** [comment]
@hitblast Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by hitblast** [reply]
@kshitij-halankar Thanks! Starring...

**Reply by number0x0001** [reply]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by aarongeo1** [reply]
@number0x0001 done https://github.com/aarongeo1/NUM-CLASSIFY

**Comment by ghost** [comment]


**Reply by hitblast** [reply]
Thanks! Done too.

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by hitblast** [reply]
Alright :D

**Comment by 41pha1** [comment]
Done
https://github.com/41pha1/MIDI-Converter

**Reply by hitblast** [reply]
@41pha1 Thanks! Done too.

**Comment by aarongeo1** [comment]
done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by hitblast** [reply]
Thanks! Done as well.

**Comment by drk1rd** [comment]
Done!
https://github.com/drk1rd/drk1rd

**Reply by hitblast** [reply]
@drk1rd Thanks! I'll be starring as well.

**Comment by bingyun123** [comment]


**Reply by hitblast** [reply]
Thanks for your response! I don't see a star though, just saying :P

**Reply by hitblast** [reply]
:D Alright then, starring as well!
